src/lib.rs

```diff
@@ -172,6 +173,21 @@ impl NotionApi {
         }
     }
 
+    pub async fn create_page<T: Into<PageCreateRequest>>(&self, page: T) -> Result<Page, Error> {
```

Comment:
Can we add a doc document to this?

```suggestion
    /// Creates a new page
    pub async fn create_page<T: Into<PageCreateRequest>>(&self, page: T) -> Result<Page, Error> {
```